docs/bugs.json

```diff
@@ -7,6 +7,14 @@
         "fixed": "0.6.8",
         "severity": "very low"
     },
+    {
+        "name": "MissingEscapingInFormatting",
+        "summary": "String literals passed to internal functions where not properly escaped when the ABIEncoderV2 was enabled.",
+        "description": "When the ABIEncoderV2 was enabled, passing a string with a certain amount of backslashes to a builtin function would lead to incorrect intermediate code generation.",
```

Comment:
``a certain amount of backslashes``? I haven't been looked into this or thought about this, but this is a bit imprecise. Isn't it in general all string literals containing escape sequences?

Reply:
No, it's more like, all odd amounts of escapes? But it's also not only that..

Reply:
I can gather all the exact cases. I didn't consider it that important yet to do that though...

Reply:
I don't think this is only about builtin functions - the regular function argument encoding is also affected. Some non-v2 code uses the routine, but only for internal revert messages, so ABIEncoderV2 is a requirement.